src/org/gk/render/RenderableComplexDrug.java

```diff
@@ -0,0 +1,20 @@
+package org.gk.render;
+
+public class RenderableComplexDrug extends RenderableComplex implements DefaultRenderConstants {
```

Comment:
No implements!

Reply:
Understood!

Reply:
Fixed in [e77f0a9](https://github.com/reactome/CuratorTool/pull/23/commits/e77f0a99bd4c6699021ff050f23d7c451e9e7d35).